pandas/core/arrays/masked.py

```diff
@@ -325,6 +327,16 @@ def take(
 
         return type(self)(result, mask, copy=False)
 
+    def isin(self, values) -> "BooleanArray":
```

Comment:
hmm we don't need an overrdie for NumericArray?

Reply:
I think we don't have to override this method in `NumericArray`. The subclass (`IntArray`, `FloatArray`)'s `isin` method works well.